src/spell.c

```diff
@@ -464,12 +464,18 @@ spell_check(
     /* Find the normal end of the word (until the next non-word character). */
     mi.mi_word = ptr;
     mi.mi_fend = ptr;
+    int camel_case = 0;
```

Comment:
Declaring a variable here is not allowed in C89 and causes compilation error on MSVC.